### PR TITLE
Extract epic GraphQL - Leon

### PR DESCRIPTION
**Pull request by Leon-JS02**
Added scripts to:
- Extract data from Epic Games' GraphQL API
- Dockerise the Epic Games extract
- Lambda Handler.
- Fixed bug in Steam tests.
- Added tests for some functions.
- Added README

BOSH

### CODE REVIEW

File: web_scraping/epic_games_scraper/README.md
Comment (mehrdadhalali) [inline web_scraping/epic_games_scraper/README.md:1]: Very nice README

File: web_scraping/epic_games_scraper/README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-## Epic Games Scraper
\ No newline at end of file
+## Epic Games Extract
+- Extracts data from Epic Games' **undocumented** GraphQL api.
```
Comment (mehrdadhalali) [inline web_scraping/epic_games_scraper/README.md:2]: Nice specification

File: web_scraping/epic_games_scraper/lambda_handler.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"""Script that runs when AWS Lambda is triggered."""
+
+from extract_epic import process_listings
+
+# pylint: disable=W0613
```
Comment (mehrdadhalali) [inline web_scraping/epic_games_scraper/lambda_handler.py:5]: What did you mean by this

Comment (Leon-JS02) [reply web_scraping/epic_games_scraper/lambda_handler.py:5]: Unused variable, lambda handlers need the two variables

**Review by mehrdadhalali — APPROVED** [review]

File: web_scraping/epic_games_scraper/push-to-ecr.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+source .env
+
+aws ecr get-login-password --region eu-west-2 | docker login --username AWS --password-stdin $AWS_ACCOUNT_ID.dkr.ecr.eu-west-2.amazonaws.com
+aws ecr create-repository --repository-name $ECR_REPO_NAME --region eu-west-2
+docker build -t $ECR_REPO_NAME . --platform "linux/amd64"
+docker tag $ECR_REPO_NAME:latest $AWS_ACCOUNT_ID.dkr.ecr.eu-west-2.amazonaws.com/$ECR_REPO_NAME:latest
+docker push $AWS_ACCOUNT_ID.dkr.ecr.eu-west-2.amazonaws.com/$ECR_REPO_NAME:latest
```
Comment (paulauus) [inline web_scraping/epic_games_scraper/push-to-ecr.sh:7]: Very useful

**Review by paulauus — APPROVED** [review]

**Pull request merged**